openedx/core/djangoapps/video_pipeline/config/waffle.py

```diff
@@ -8,10 +8,34 @@
 # Videos Namespace
 WAFFLE_NAMESPACE = 'videos'
 
-# Waffle flag telling whether youtube is deprecated.
+# .. toggle_name: videos.deprecate_youtube
+# .. toggle_implementation: WaffleFlag
+# .. toggle_default: False
+# .. toggle_description: Waffle flag telling whether youtube is deprecated. When
+#   enabled, videos are no longer uploaded to YouTube as part of the video pipeline.
+# .. toggle_category: videos
+# .. toggle_use_cases: open_edx
+# .. toggle_creation_date: 2018-08-03
+# .. toggle_expiration_date: None
+# .. toggle_warnings: None
+# .. toggle_tickets: https://github.com/edx/edx-platform/pull/18765
+# .. toggle_status: supported
 DEPRECATE_YOUTUBE = 'deprecate_youtube'
+# .. toggle_name: videos.enable_devstack_video_uploads
+# .. toggle_implementation: WaffleFlag
+# .. toggle_default: False
+# .. toggle_description: When enabled, use Multi-Factor Authentication (MFA) for
+#   authenticating to AWS. These short-lived access tokens are well suited for
+#   development (probably?).
```

Comment:
1. Could you add review comments with any questions or concerns you have? Not sure if this was copied from somewhere, or if it is your question.
2. Should we be removing status from all of these? If we would lose any info, just include it elsewhere.
3. What is our recommendation for empty annotations. Is it possible to leave them empty. If they must have a value, I don’t love None. Maybe n/a? I know this is only relevant when looking at the code.

Reply:
> Could you add review comments with any questions or concerns you have?

Sure thing. I'll add my questions in the description of this PR as I go along.

> Should we be removing status from all of these?

Yes -- but I'd like to get the code-annotation PR merged before I update the annotations to the new format. I'll assume all feature toggles are unsupported, and for those that are not I'll add a warning.

> What is our recommendation for empty annotations. Is it possible to leave them empty. If they must have a value, I don’t love None. Maybe n/a? I know this is only relevant when looking at the code.

Just remove empty annotations. For warnings, "None" and "N/A" at all assumed to be `None` (see [this piece of code](https://github.com/edx/code-annotations/blob/f61565ddd150dbf8b96fc66de806ae0253bc951c/code_annotations/config_and_tools/sphinx/extensions/featuretoggles.py#L118))